lib/actions/api.js

```diff
@@ -720,6 +823,7 @@ window.setInterval(() => {
  */
 
 function createQueryAction (endpoint, responseAction, errorAction, options = {}) {
+  // eslint-disable-next-line complexity
```

Comment:
Please remove this eslint-disable, but fixing isn't necessary in this PR. This is an instance where it'd be a good idea to eventually refactor.

Reply:
Will refactor now since this is scary.